packages/loader/container-loader/src/container.ts

```diff
@@ -1293,7 +1297,10 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
             values);
 
         // The load context - given we seeded the quorum - will be great
-        await this.createDetachedContext(attributes);
+        await this.instantiateContextDetached(
+            false, // existing
```

Comment:
Just to make sure I understand next steps: Not using this._existing here assumes that you have a plan to remove this._existing. Without that, I'd not keep using it to avoid drift in the code where these things are not aligned.

Reply:
Yes, that is the plan.